.github/workflows/sanity_checks.yml

```diff
@@ -196,6 +196,10 @@ jobs:
           python3 -m pip install license-expression
           python3 -m pip install --pre meson
 
+      - name: Work around runner-images issue 9966
+        run: |
+          brew list -1 | grep python | while read formula; do brew unlink $formula; brew link --overwrite $formula; done
```

Comment:
This seems a bit long as a one-liner.  Any reason not to keep the original formatting?

Reply:
Just because that's what it seemed others were doing. I don't know YAML and formatting details well enough to know if the multi-line version would work or not without testing both.

I suppose I can try and see if it helps?